### PR TITLE
Split integration documentation

### PR DESCRIPTION
**Pull request by jopdeklein**

### CODE REVIEW

File: docs/dead-code-removal.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+The codemods use different strategies to clean losing toggles and keep the
+winners:
+
+- winning values are kept
+- for functions, the function body is kept
+
+For the losing boolean toggles and losing multi toggle variants:
+
+- losing toggles are pruned
+- if the losing side is an JSXExpression, we clean it up including the variables
```
Comment (Gerrit88) [inline docs/dead-code-removal.md]:
```suggestion
- if the losing side is a JSXExpression, we clean it up including the variables
```

File: docs/dead-code-removal.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+  that are referenced from there
+
+The codemods are designed to prune toggles that are null, allowing you to
+execute code only for one variant of an multi toggle experiment.
```
Comment (Gerrit88) [inline docs/dead-code-removal.md]:
```suggestion
execute code only for one variant of a multi toggle experiment.
```

**Review by Gerrit88 — COMMENTED** [review]
Nice! This definitely makes the documentation easier to read :+1: 
Some nitpicking suggestions below.

**Review by votsa — APPROVED** [review]

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (jopdeklein) [inline LICENSE:1]: Fixes #4

File: README.md [hunk 4]
```diff
@@ -48,57 +48,19 @@ the one you need, for now the "simple" is the default and the Optimizely adapter
...
-  that are referenced from there
-
-The codemods are designed to prune toggles that are null, allowing you to
-execute code only for one variant of an multi toggle experiment.
-
-## Initialization
-
-TODO: This section describes both integrations, split when converting to
-monorepo.
+## Integrations
```
Comment (jopdeklein) [inline README.md:51]: Fixes #3

File: docs/dead-code-removal.md [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+- losing toggles are pruned
+- if the losing side is a JSXExpression, we clean it up including the variables
+  that are referenced from there
+
+The codemods are designed to prune toggles that are null, allowing you to
+execute code only for one variant of a multi toggle experiment.
+
+## Running the codemods
+
+There two codemods supplied with Opticks, one for Boolean Toggles, one for Multi Toggles, they can be found in the `src/transform` directory.
```
Comment (jopdeklein) [inline docs/dead-code-removal.md]: Fixes #2

**Pull request merged**